swagger.json

```diff
@@ -1055,6 +1055,32 @@
                         }
                     }
                 }
+            },
+            "post": {
+                "tags": [
+                    "Fleet","Default"
+                ],
+                "summary": "/fleet/maintenance/dvirs",
+                "description": "Create a minimal new dvir, marking a vehicle safe or unsafe.",
```

Comment:
Should we describe that mechanics are not required to submit all DVIR related fields?

Reply:
Rewording to `Create a new dvir, marking a vehicle or trailer safe or unsafe.` 
Do we normally specify which fields are/aren't required in descriptions? I thought swagger's layout handled that.

Reply:
Ah yes, drivers have one field that's required that mechanics don't have - specifying if a DVIR is pre- or post-trip. That field isn't included yet though, since mechanics don't use it.